### DISCUSSION TITLE
当在开放平台设置完毕后，如何测试配置是否成功？

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#3020] [Developer]
# 为什么需要测试

> 因为你的数据都在你的同步盘（坚果云 or Dropbox），简悦是无法获取到你的数据的，所以需要在 [开通开发平台](https://github.com/Kenshin/simpread/discussions/2757)

> 在授权或配置时，难免会出现错误，因为涉及到多个服务，所以排查相对有难度，为了降低难度，可以利用下面的方式测试你的配置是否成功。

# 测试前提

> 测试此方式的前提是你已经进入了开放平台，请复制 Token 到你剪切板。（下面的测试会使用它）

> 需要勾选快照功能，如何勾选 [请看这里](https://github.com/Kenshin/simpread/discussions/2994)。

![image](https://user-images.githubusercontent.com/81074/140601704-53ec5538-3a3d-405b-bf14-e52044642e42.png)

# 测试

> 请在浏览器打开 [此链接](https://hoppscotch.io/?v=1&method=POST&endpoint=https://api-wrap.simpread.pro/api/service/webhook/***&body=%7B%22body%22:%22%7B%20%5C%22url%5C%22:%20%5C%22https://sspai.com/post/67074%5C%22%20%7D%22,%22contentType%22:%22text/plain%22%7D) 并根据下图所示进行测试

![image](https://user-images.githubusercontent.com/81074/140601869-ead234b5-2299-495d-bd7a-ddd59d902708.png)

> 如果出现 `201` 则说明 API 无误，接下来需要验证你的同步盘（假设在坚果云中），根据下图所示的验证即可。

> 如果是其它值，也对照你的同步盘与下图的区别，以及 [各个返回值说明](https://github.com/Kenshin/simpread/discussions/3020#discussioncomment-1598514)。

![image](https://user-images.githubusercontent.com/81074/140601992-864c1bba-d9df-4383-915a-640426f0ee0c.png)

![image](https://user-images.githubusercontent.com/81074/140602060-ddff2679-dea2-44e0-a50c-21393768e0ed.png)

> 一般情况下使用此功能都是使用了 [同步助手的自动同步](https://github.com/Kenshin/simpread/discussions/2754) 功能，看下面的选项是否正确（位置在：选项页 → 共通 → 自动同步）

![image](https://user-images.githubusercontent.com/81074/140602124-be6cd6b2-8578-44fc-92b7-09bc21111928.png)

> 如果上述均没问题的话，打开稍后读，会看到这篇文章

![image](https://user-images.githubusercontent.com/81074/140602269-a9d638c5-eb23-4ae7-966c-9dc07c27eadd.png)

# 关联阅读

- https://github.com/Kenshin/simpread/discussions/2757
- https://github.com/Kenshin/simpread/discussions/1570
- https://github.com/Kenshin/simpread/discussions/2959
- https://github.com/Kenshin/simpread/discussions/2974

**Comment by Kenshin** [comment]
# 返回值的含义

| 返回值 | 含义                                                         |
| ------ | ------------------------------------------------------------ |
| 201    | 返回正常（稍后读 / 快照）均已经保存正常                      |
| -1     | 服务器错误，一般来说稍后再测试。（极少会出现此问题）         |
| 401    | 没有开通开放平台，请 [先开通](https://github.com/Kenshin/simpread/discussions/2757)  |
| 404    | 没有使用授权坚果云 / Dropbox 给简悦，如何授权 [请看这里](https://github.com/Kenshin/simpread/discussions/1570) 或你的同步盘里面没有 `output` 文件夹，[细节看这里](https://github.com/Kenshin/simpread/discussions/2956) |

> 同步盘的意思是：你的坚果云或 Dropbox

**Comment by DanShui-Yu** [comment]
返回504是什么意思？

**Reply by Kenshin** [reply]
需要详细说明你的步骤 + 截图
